extensions/opus/src/main/jni/opus_jni.cc

```diff
@@ -59,6 +59,7 @@ jint JNI_OnLoad(JavaVM* vm, void* reserved) {
 }
 
 static const int kBytesPerSample = 2;  // opus fixed point uses 16 bit samples.
+static const int kMaxOpusOutputPacketSizeSamples = 960 * 6;// Maximum packet size used in Xiph's opusdec.
```

Comment:
nit: two spaces after the semi-colon. actually the comment can go since there's enough justification for this line in the commit message.